packages/merkle-distributor/src/merkleTree.js

```diff
@@ -0,0 +1,131 @@
+const web3 = require("web3");
+const ethereumjsUtil = require("ethereumjs-util");
+
+class MerkleTree {
+  constructor(elements) {
```

Comment:
Could you describe what type elements is?

Reply:
I'll be sure to add better comments in the SDK. Elements is an array of hashed leaves.